src/components/forms/Radio/Radio.tsx

```diff
@@ -12,6 +12,8 @@ interface RadioProps {
     | React.RefObject<HTMLInputElement>
     | null
     | undefined
+  tile?: boolean
+  labelDescription?: React.ReactNode
```

Comment:
I think this just needs to be a `string`

Reply:
My thinking was that the label may end up having formatting, links, or other inline html. Would that still be possible if it's a `string`?

Reply:
Yea, you're right. People may want to at least bold some of the text in the label and things like that. Most of my hesitancy came from [`<span>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span) taking  "[phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#phrasing_content)" only. That does in include a bunch more things than just text, so we can leave it at `ReactNode` and the onus will be on the implementer to choose the right elements.